js-api-doc/value/color.d.ts

```diff
@@ -1,5 +1,83 @@
+import {List} from 'immutable';
+
 import {Value} from './index';
 
+/** The HSL color space name. */
+export type ColorSpaceHSL = 'hsl';
+
+/** The HSL color space channel names. */
+export type ChannelNameHSL = 'hue' | 'saturation' | 'lightness';
+
+/** The HWB color space name. */
+export type ColorSpaceHWB = 'hwb';
+
+/** The HWB color space channel names. */
+export type ChannelNameHWB = 'hue' | 'whiteness' | 'blackness';
+
+/** The Lab / Oklab color space names. */
+export type ColorSpaceLab = 'lab' | 'oklab';
+
+/** The Lab / Oklab color space channel names. */
+export type ChannelNameLab = 'lightness' | 'a' | 'b';
+
+/** The LCH / Oklch color space names. */
+export type ColorSpaceLCH = 'lch' | 'oklch';
+
+/** The LCH / Oklch color space channel names. */
+export type ChannelNameLCH = 'lightness' | 'chroma' | 'hue';
+
+/** Names of color spaces with RGB channels. */
+export type ColorSpaceRGB =
+  | 'a98-rgb'
+  | 'display-p3'
+  | 'prophoto-rgb'
+  | 'rgb'
+  | 'srgb'
+  | 'srgb-linear';
+
+/** RGB channel names. */
+export type ChannelNameRGB = 'red' | 'green' | 'blue';
+
+/** Names of color spaces with XYZ channels. */
+export type ColorSpaceXYZ = 'xyz' | 'xyz-d50' | 'xyz-d65';
+
+/** XYZ channel names. */
+export type ChannelNameXYZ = 'x' | 'y' | 'z';
+
+/** All supported color space channel names. */
+export type ChannelName =
```

Comment:
It seems like this should also include `'alpha'` (or maybe each individual space's `ChannelName` should include it, since `'alpha'` is a valid channel for every space).